cstore_fdw.c

```diff
@@ -358,7 +361,7 @@ CStoreProcessUtility(Node * parseTree, const char *queryString,
 								  destReceiver, completionTag);
 		}
 
-		TruncateCStoreTables(cstoreTablesList);
+		TruncateCStoreTables(cstoreRelationList);
```

Comment:
Sorry, I‌ had missed that. Maybe we should close outside `TruncateCStoreTables()` as you suggested so future readers of code don't miss that.